app/src/main/java/com/opsat7373/moviestars/presentation/fragment/MovieDetailsFragment.kt

```diff
@@ -0,0 +1,56 @@
+package com.opsat7373.moviestars.presentation.fragment
+
+import android.os.Bundle
+import android.util.Log
+import androidx.fragment.app.Fragment
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.databinding.ObservableBoolean
+import androidx.lifecycle.ViewModelProvider
+import com.opsat7373.moviestars.databinding.FragmentMovieDetailsBinding
+import com.opsat7373.moviestars.presentation.viewmodel.MovieDetailsViewModel
+
+/**
+ * A simple [Fragment] subclass.
+ * Use the [MovieDetailsFragment.newInstance] factory method to
+ * create an instance of this fragment.
+ */
+class MovieDetailsFragment : Fragment() {
+
+    private lateinit var movieDetailsViewModel : MovieDetailsViewModel
+
+    private val isLoading : ObservableBoolean = ObservableBoolean(true)
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        movieDetailsViewModel = ViewModelProvider(this).get(MovieDetailsViewModel::class.java)
+
+        val movieId : Int? = arguments?.getInt("movieId")
+        if (movieId != null) {
```

Comment:
can the page work without movieId? If no, it's better to throw NPE (just use !!. instead of ?. )